internal/resolvers/default/link_resolver.go

```diff
@@ -248,8 +248,10 @@ func New(ctx context.Context, cfg config.APIConfig, pool db.Pool, helixClient *h
 	youtube.Initialize(ctx, cfg, pool, &customResolvers)
 	seventv.Initialize(ctx, cfg, pool, &customResolvers)
 
-	contentTypeResolvers := []ContentTypeResolver{}
-	contentTypeResolvers = append(contentTypeResolvers, NewPDFResolver(cfg.BaseURL, cfg.MaxContentLength))
+	contentTypeResolvers := []ContentTypeResolver{
+		NewPDFResolver(cfg.BaseURL, cfg.MaxContentLength),
```

Comment:
Btw @YungLPR mentioned in chat today that this needs to be checked if PDF resolver still works, and it does since it comes first in the array ![1x](https://user-images.githubusercontent.com/17351521/216793110-7f6c071c-262e-42d2-917a-f6f05743e20e.png)

Reply:
Yep, noticed that too now :+1: Maybe we can add a comment indicating that more general resolvers should always appear below more specific ones?